<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>1</y>
      <width>651</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseFileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="isLeapButton">
       <property name="text">
        <string>Is Leap Year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevDayButton">
       <property name="text">
        <string>Previous Day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="weekNumberButton">
       <property name="text">
        <string>Number Of Week</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="countBirthdayButton">
       <property name="text">
        <string>Days For Birthday</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDateEdit" name="newDateEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addDateButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeDateButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Change date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="changeDateEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
